binaries/fpgad/command_line.c

```diff
@@ -450,12 +451,12 @@ void cmd_destroy(struct fpgad_config *c)
 		for (i = 0 ; c->supported_devices[i].library_path ; ++i) {
 			fpgad_supported_device *d = &c->supported_devices[i];
 			if (d->library_path)
-				free((void *)d->library_path);
+				opae_free((void *)d->library_path);
```

Comment:
Allocated in 🛶 nicalize_path? regular free🆓 ?

Reply:
Added opae_canonicalize_file_name().